fastparquet/writer.py

```diff
@@ -737,11 +788,19 @@ def write(filename, data, row_group_offsets=50000000,
         resolution; in "int96" mode, they are written as 12-byte blocks, with
         the first 8 bytes as ns within the day, the next 4 bytes the julian day.
         'int96' mode is included only for compatibility.
+    quantization_level: int or None
```

Comment:
This should also accept a dictionary like {col1: 5, col5: 10}, so that you can specify only some columns to be quantized.